src/actions/remoteConfigActions.js

```diff
@@ -41,3 +47,28 @@ export const loadRemoteConfigAction = () => {
       .catch(e => { reportLog('Failed to fetch feature flags or initialize with defaults', e); });
   };
 };
+
+export const setUserPropertiesAction = () => (
+  dispatch: Dispatch,
+  getState: GetState,
+) => {
```

Comment:
This action seems to be about analytics, and hence would be better to put it into `analyticsActions` file.